wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift

```diff
@@ -95,6 +95,13 @@ public enum SemanticColors {
         static let emojiCategoryDefault = UIColor(light: Asset.Colors.gray80, dark: Asset.Colors.gray60)
         static let emojiCategorySelected = UIColor(light: Asset.Colors.black, dark: Asset.Colors.white)
 
+        // The init here is different because in light mode we would like the color of the border
+        // to be clear. The initializer in all other cases in this file expects a type of ColorAsset
+        // in both light and dark mode.
+        static let borderMutedNotifications = UIColor { traits in
+            traits.userInterfaceStyle == .dark ? Asset.Colors.gray70.color : .clear
+        }
```

Comment:
Can't we have `Asset.Colors.clear`?

Reply:
I didn't want to add a clear color in our Color.xcassets. I would much rather use it directly from UIColor. That's why I created a different init for that color.

Reply:
I think it's simpler to understand to use the Apple framework clear color instead of custom ones.